SConstruct

```diff
@@ -106,23 +106,15 @@ linker_to_flags = {
   'clang++'  : clang_linker_flags
 }
 
-
-def cuda_installation():
+def cuda_installation(env):
```

Comment:
For consistency, instead of receiving an `env` here, could we take the name of the CUDA compiler, in the same way that `omp_installation` receives the name of the `CXX` compiler?